src/web/components/Input/RadioInput.stories.tsx

```diff
@@ -1,19 +1,30 @@
 import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { FormProvider, useForm } from 'react-hook-form';
 
-import { Form } from '../Core/Form';
 import { RadioInput } from './RadioInput';
 import { Option } from './SelectInput';
 
+import '../Core/Form.scss';
```

Comment:
Decided to leave the scss file since it is used several places but let me know if you think I should rename or move it

Reply:
I think src/web/styles would be a better place for it now.

Reply:
Moved & renamed